samples/fileChangeStorageClass.js

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// sample-metadata:
+//   title: Change File's Storage Class.
+//   description: Change File's Storage Class.
+//   usage: node fileChangeStorageClass.js <BUCKET_NAME> <FILE_NAME> <CLASS_NAME>
+
+function main(
+  bucketName = 'my-bucket',
+  filename = 'file.txt',
+  storageClass = 'STANDARD'
+) {
+  // [START storage_change_file_storage_class]
+  // Imports the Google Cloud client library
+  const {Storage} = require('@google-cloud/storage');
+
+  // Creates a client
+  const storage = new Storage();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const bucketName = 'Name of a bucket, e.g. my-bucket';
+  // const filename = 'File to access, e.g. file.txt';
+  // const storageClass = 'Name of a storage class, e.g. STANDARD';
+  async function fileChangeStorageClass() {
+    await storage.bucket(bucketName).file(filename).setMetadata({
+      storageClass,
+    });
```

Comment:
I get `Error: Invalid argument` Error with this set up. It seems that existing file's storageClass does not change.
I believe [`file#setStorageClass()`](https://github.com/googleapis/nodejs-storage/blob/master/src/file.ts#L3451) function should be used, which accomplishes setting a different `storageClass` via `file#copy`